### PR TITLE
Add table 5 filtering

### PR DESCRIPTION
**Pull request by amiersmith**

### CODE REVIEW

File: frontend/src/data/nsf23313_2_1/dcids/values.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    'Count_Person_Citizen_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_NotHispanicOrLatino_Male_PostDoctoralAppointeeOccupation_TwoOrMoreRaces',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree_UnknownEthnicity_Male_UnknownRace',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_MastersDegree_UnknownEthnicity_Male_UnknownRace',
+    'Count_Student_Citizen_ScienceOrEngineeringOrHealth_DoctorateDegree_UnknownEthnicity_Male_UnknownRace',
+    'Count_Person_Citizen_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_UnknownEthnicity_Male_PostDoctoralAppointeeOccupation_UnknownRace',
+    'Count_Student_VisaHolder_ScienceOrEngineeringOrHealth_MastersDegreeOrDoctorateDegree_UnknownEthnicity_Male',
+    'Count_Student_VisaHolder_ScienceOrEngineeringOrHealth_MastersDegree_UnknownEthnicity_Male',
+    'Count_Student_VisaHolder_ScienceOrEngineeringOrHealth_DoctorateDegree_UnknownEthnicity_Male',
+    'Count_Person_VisaHolder_EducationalAttainmentDoctorateDegree_ScienceOrEngineeringOrHealth_UnknownEthnicity_Male_PostDoctoralAppointeeOccupation',
```
Comment (isaac-looker) [inline frontend/src/data/nsf23313_2_1/dcids/values.ts:97]: It seems like we're missing Female Visaholders.

File: frontend/src/data/nsf23313_2_1/categories/values.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ]),
+    'education': new Set<string>([
+        'DoctorateDegree',
+        'MastersDegree',
+    ]),
+    'citizenship': new Set<string>([
+        'Citizen',
+        'VisaHolder',
+    ]),
+    'occupation': new Set<string>([
```
Comment (isaac-looker) [inline frontend/src/data/nsf23313_2_1/categories/values.ts:25]: I think these are each their own category. See [here](file:///home/drumsound/Downloads/nsf22319-tab002-001.pdf) for details. The sum under each column add to 100%

**Review by isaac-looker — CHANGES_REQUESTED** [review]

File: frontend/src/data/nsf23313_2_1/categories/values.ts (hunk 2 above)
Comment (amiersmith) [reply frontend/src/data/nsf23313_2_1/categories/values.ts:25]: separated the categories

File: frontend/src/data/nsf23313_2_1/dcids/values.ts (hunk 1 above)
Comment (amiersmith) [reply frontend/src/data/nsf23313_2_1/dcids/values.ts:97]: added

File: frontend/src/data/nsf23313_2_1/queries/values.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+// If we're querying both
+// 1. a subset of enthicity dimensions, and
+// 2. all gender dimensions
+// we need to query each gender dimension separately and the sum the values; that is,
+// we can't use the `gender:` dimension key to search for dcids that have a summarized
+// ethnicity metric that includes all genders.
+// EXAMPLE:
+// const categoryDependencies: [string, string][] = [['ethnicity', 'gender']]
+const categoryDependencies: [string, string][] = [['ethnicity', 'citizenship'], ['race', 'citizenship']]
```
[the remark this answers is not in the document]
Comment (amiersmith) [reply frontend/src/data/nsf23313_2_1/queries/values.ts]: good catch, done

**Pull request closed without merging**